app/src/main/java/com/github/swent/echo/compose/association/AssociationSubscriptionsScreen.kt

```diff
@@ -1,50 +0,0 @@
-package com.github.swent.echo.compose.association
-
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.collectAsState
-import androidx.compose.runtime.getValue
-import androidx.compose.runtime.setValue
-import androidx.compose.ui.res.stringResource
-import com.github.swent.echo.R
-import com.github.swent.echo.ui.navigation.NavigationActions
-import com.github.swent.echo.viewmodels.user.UserProfileViewModel
-
-// This screen is a list of the subscriptions of a user to associations
-@Composable
-fun AssociationSubscriptionsScreen(
```

Comment:
I believe this screen should be kept because it is used to unfollow associations easily and have an overview of the followed associations of the user.

Reply:
The overview is available on the AssociationMainScreen and massive unfollow is a pretty rare thing. It also avoid unintentional unfollow and drastically simplify the code + user flow (so we can use ListAssociation without trouble in other pages, which is used in the future also in AssociationSearch, without doing weird things with the UI to not have follow button on comitteeList or when we display all associations)

Reply:
I though that last week we decided to keep the mass unfollow screen since I added a snackbar to undo a miss click, but I must have misunderstood the final decision.